src/index.js

```diff
@@ -4,34 +4,33 @@ import { Provider } from 'react-redux';
 import { createStore } from 'redux';
 import { BrowserRouter, Route, Switch } from 'react-router-dom';
 
-import './index.css';
-
-import Search from './components/Search.js';
-import Article from './components/Article.js';
-import Author from './components/Author.js';
-import Reader from './components/Reader.js';
-import Home from './components/Home.js';
+import './assets.js';
 
+import Search from './components/search';
+import Article from './components/article';
+import Author from './components/author';
+import Reader from './components/reader';
+import Home from './components/home';
 import registerServiceWorker from './registerServiceWorker';
 
 import reducers from './reducers';
 
-let store = createStore(reducers);
+let store = createStore(reducers)
 
 ReactDOM.render(
-    <Provider store={ store }>
-      <BrowserRouter>
-        <div>
-          <Switch>
-            <Route path="/search/" component={ Search } />
-            <Route path="/Article/:id" component={ Article } />
-            <Route path="/Author/:id" component={ Author } />
-            <Route path="/Reader/:id" component={ Reader } />
-            <Route path="/" component={ Home } />
-          </Switch>
-        </div>
-      </BrowserRouter>
-    </Provider>
-    , document.getElementById('root'));
+  <Provider store={ store }>
+    <BrowserRouter >
+      <div>
+        <Switch>
+          <Route path="/search/" component={ Search } />
+          <Route path="/Article/:id" component={ Article } />
+          <Route path="/Author/:id" component={ Author } />
+          <Route path="/Reader/:id" component={ Reader } />
+          <Route path="/" component={ Home } />
+        </Switch>
+      </div>
+    </BrowserRouter>
+  </Provider>
```

Comment:
I made a mistake here. No spaces needed for braces in JSX.
https://github.com/airbnb/javascript/blob/master/react/README.md#spacing